### ISSUE TITLE
A user should be able to navigate through pages of search results

### ISSUE THREAD

**Issue by andrewmaier** [18F/afsbirez#184]
The ux for this is a bit wonky, regardless of whether or not they use a keyboard:

- We should return users to the top of the page after clicking "next"
- We should specify what results are being shown at the top of the page (for example: 11-20 of 52)
- We should update the URL bar so users can share a specific page of results.

I can craft some nicer header styles for displaying the currently displayed results alongside the header

**Comment by DavidEBest** [comment]
I'm all for this. :+1: Go ahead and mock up some header styles.

**Comment by andrewmaier** [comment]
On it this afternoon :)
